plugins/builddep.py

```diff
@@ -157,7 +166,10 @@ def _src_deps(self, src_fn):
                 logger.error("Error: public key not available, add "
                              "'--nogpgcheck' option to ignore package sign")
                 return
-            raise
+            elif e[0] == 'error reading package header':
```

Comment:
have you bothered testing this?
(https://bugzilla.redhat.com/show_bug.cgi?id=1218299)